src/js/midi/midi.js

```diff
@@ -97,7 +95,7 @@ class MIDI extends EventEmitter {
           const cmd = data >> 4
           const channel = data & 0x0f
 
-          if (device.channels[channel] && device.channels[channel].enabled) {
+          if (device.channels[channel]?.enabled === true) {
```

Comment:
```suggestion
          if (device.channels[channel].enabled === true) {
```

Reply:
I think the optional chaining operator is okay here - am I missing something?

Reply:
Oh, that's an actual feature and not a typo? Never encountered this before, carry on.

Reply:
No worries! It's a nifty Javascript operator - it doesn't execute the code to the right of the question mark if the object it's called on is null or undefined.

Reply:
Yes, it's the optional chaining operator. To read it simply, if the part before the question mark is undefined, then the right part will not be executed and the whole expression will be undefined. It's useful for protecting yourself agains errors, like `undefined is not an object`.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining